app-dicts/mecab-ipadic/mecab-ipadic-2.7.0.20070801.ebuild

```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2022 Gentoo Authors
+# Copyright 1999-2023 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI="6"
+EAPI="8"
 
 MY_P="${PN}-${PV%.*}-${PV/*.}"
```

Comment:
(Can't comment on right line.)

While here, please check if mecab is an RDEPEND too...

Reply:
Will look into it. For testing that, is unmerging mecab and seeing if that runs considered good practice?